src/storage-types/src/sources.rs

```diff
@@ -68,6 +71,38 @@ pub use crate::sources::postgres::PostgresSourceConnection;
 
 include!(concat!(env!("OUT_DIR"), "/mz_storage_types.sources.rs"));
 
+/// A fraternal twin of [`UnresolvedItemName`] that can implement [`Arbitrary`].
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct ExportReference(pub Vec<Ident>);
+
+impl proptest::prelude::Arbitrary for ExportReference {
+    type Parameters = ();
+    type Strategy = BoxedStrategy<Self>;
+
+    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
+        // Generate a set of valid `Ident` strings.
+        let string_strategy = string_regex("^[a-zA-Z_][a-zA-Z0-9_$]{3,10}$").unwrap();
```

Comment:
it'd be nice if we generated at least 1 character that needed to be escaped! but not necessary